app/templates/vendor/new_csv.html

```diff
@@ -0,0 +1,111 @@
+{% extends 'layouts/base.html' %}
+{% import 'macros/form_macros.html' as f %}
+
+{% block scripts %}
+{% endblock %}
+
+{% block content %}
+<div class="ui stackable centered grid container">
+    <div class="six wide column">
+        <a class="ui basic compact button" href="{{ url_for('vendor.index') }}">
+            <i class="caret left icon"></i>
+            Back to dashboard
+        </a>
+
+        <h2 class="ui header">
+            Upload CSV
+        </h2>
+
+        {% set flashes = {
+        'error': get_flashed_messages(category_filter=['form-error']),
+        'warning': get_flashed_messages(category_filter=['form-check-email']),
+        'info': get_flashed_messages(category_filter=['form-info']),
+        'success': get_flashed_messages(category_filter=['form-success'])
+        } %}
+
+        {{ f.begin_form(form, flashes) }}
+
+        {{ f.render_form_field(form.file_upload) }}
+
+
+        {{ f.form_message(flashes['error'], header='Something went wrong.', class='error') }}
+        {{ f.form_message(flashes['warning'], header='Check your item information.', class='warning') }}
+        {{ f.form_message(flashes['info'], header='Information', class='info') }}
+        {{ f.form_message(flashes['success'], header='Success!', class='success') }}
+
+        {% for field in form | selectattr('type', 'equalto', 'SubmitField') %}
+        {{ f.render_form_field(field) }}
+        {% endfor %}
+
+        {{ f.end_form() }}
+    </div>
+    <div class="six wide column">
+        <table id="listing_table" class="ui celled table">
+            <thead>
+            <tr>
+                <th>Item Name</th>
+                <th>Price</th>
+                <th>Description</th>
+                <th>Category</th>
+                <th>Status</th>
+            </tr>
+            </thead>
+            <tbody>
+            {% for listing in listings %}
+                {% if listing.updated == 0 %}
+                    <tr class="ui inverted blue table">
+                        <td>{{listing.name}}</td>
+                        <td>{{listing.price}}</td>
+                        <td>{{listing.description}}</td>
+                        <td>{{listing.Category}}</td>
+                        <td><i class="icon refresh "></i> Updated Listing</td>
+                    </tr>
+
+                {% elif listing.updated == 1 %}
+                    <tr class="ui inverted green table">
+                        <td>{{listing.name}}</td>
+                        <td>{{listing.price}}</td>
+                        <td>{{listing.description}}</td>
+                        <td>{{listing.Category}}</td>
+                        <td><i class="icon checkmark "></i> Submitted New Listing</td>
+                    </tr>
+                {% elif listing.updated == 2 %}
+                    <tr class="ui inverted red table">
```

Comment:
for now yes, steven said he wanted it. i'll try to talk him out of it and then i'll change it once i do so.